backend/src/main/scala/bloop/logging/BloopLogger.scala

```diff
@@ -56,13 +57,13 @@ final class BloopLogger(
   }
 
   private def print(msg: String, fn: String => Unit): Unit = {
-    val lines = msg.split("\\r?\\n", -1)
+    val lines = msg.splitLines // reliable line splitter
     val printFn = if (colorOutput) fn else (str: String) => fn(BloopLogger.stripColors(str))
     lines.foreach(printFn)
   }
 
   private def printInfo(line: String): Unit = {
-    out.println(line)
+    out.print(line + lineSeparator) // end-of-line matches shell environments.
```

Comment:
```suggestion
    out.print(line + lineSeparator)
```

Reply:
Are you saying we should revert this:
```Scala
  private def print(msg: String, fn: String => Unit): Unit = {
    val lines = msg.splitLines // reliable line splitter
```
to this?
```Scala
  private def print(msg: String, fn: String => Unit): Unit = {
    val lines = msg.split(System.lineSeparator)
```

Reply:
It's enough to remove the comment, we should leave `msg.splitLines`.